src/cr/cube/matrix/assembler.py

```diff
@@ -50,19 +50,17 @@ def column_display_order(cls, dimensions, second_order_measures, format):
         # --- form consistent with `.row_display_order()` and we'll elaborate this when
         # --- we add sort-by-value to columns.
         collation_method = dimensions[1].order_spec.collation_method
+        # fmt: off
```

Comment:
!dm black 😢 

For my curiosity, there's a native switch statement in newer versions of python right? How far off are we from being able to use it?

Reply:
yes there's the `match` statement that is introduced in 3.10, we could use that but we need to drop 3.8 and 3.9 support, that is feasible to me considering that exporter works on 3.11 now, but dunno if cube is used even directly in other lib like pycrunch or scrunch, so I'd live it with this horrible thing for now :D ... and yes !dm black